sdk/cosmos/azure-cosmos/azure/cosmos/_retry_utility.py

```diff
@@ -76,6 +78,12 @@ def Execute(client, global_endpoint_manager, function, *args, **kwargs):
     timeout_failover_retry_policy = _timeout_failover_retry_policy._TimeoutFailoverRetryPolicy(
         client.connection_policy, global_endpoint_manager, *args
     )
+    if args and len(args) > 3:
```

Comment:
What's `args[3]`?
Given that `*args` is already being passed into all the existing policies above - I'm assuming we're not adding any new args here?
After reading the code all the way through - I'm guessing `args[3]` is the HttpRequest.
I feel this code could use a cleaner refactor to avoid just passing args[3] everywhere.....

Reply:
+1 - I would bind it with a variable with a functionally relevant name.

Reply:
@jeet1995 I agree, that's what I will do.